tests/components/aquacell/test_coordinator.py

```diff
@@ -0,0 +1,85 @@
+"""Test the Aquacell coordinator."""
```

Comment:
I personally dislike `test_coordinator.py` tests as they assume you use a coordinator. But they all look like general integration runtime things so they could look good in `test_init.py`

Reply:
Moved them to test_init.py